.github/ct-install-config/common-ct-lint.yaml

```diff
@@ -1,6 +1,6 @@
 remote: origin
 target-branch: master
-helm-extra-args: --timeout 600s --debug
+helm-extra-args: --debug
```

Comment:
Curious, not required anymore ?

Reply:
Was never valid option, just didnt throw an error before!